wpengine/faustjs
# MERL-741 - Docs: Tutorial on Third Party Block Creation

## Tasks


## Description



Documentation tutorial on how to create a block from a Third Party Plugin.

## Related Issue(s):



## Testing



1. Run the docs server
2. Navigate check a new sidebar exists inside `gutenberg` option about tutorials.
3. Check that the page renders correctly.

## Screenshots


<img width="1790" alt="Screenshot 2023-02-16 at 13 03 10" src="https://user-images.githubusercontent.com/328805/219372767-0a37a939-a233-435e-bc36-231afaa0617d.png">

## Documentation Changes



A new documentation page in `docs/gutenberg/tutorial/create-a-block-from-third-party`

## Dependant PRs
